src/fitting.jl

```diff
@@ -13,15 +13,13 @@ function fit_overpotential(model::KineticModel, k; kT=.026, kwargs...)
         storage .= compute_k(V, model; kT=kT, kwargs...) .- k
     end
 
-    # this isn't quite right yet...
-    # function grad!(storage, V)
-    #     gs = gradient(V -> compute_k(V, model; kT=kT, kwargs...) .- k, V)
-    #     for i in 1:length(V)
-    #         storage[i] = gs[i]
-    #     end
-    # end
-    # Vs = nlsolve(compare_k!, grad!, repeat([0.1], length(k)))
-    Vs = nlsolve(compare_k!, repeat([0.1], length(k)))
+    function grad!(storage, V)
+        gs = gradient(V -> sum(compute_k(V, model; kT=kT, kwargs...) .- k), V)[1]
```

Comment:
`gs` is a 1-tuple, and `gs[1]` corresponds to `V` here.

The gradients for `compute_k` should be fine here I think, but curious that `nlsolve` doesn't converge. It shouldn't need anything special I don't think. There might be some options that we can tweak.